python_modules/dagster/setup.py

```diff
@@ -74,12 +74,12 @@ def get_version() -> str:
         "alembic>=1.2.1,!=1.6.3,!=1.7.0",
         "croniter>=0.3.34",
         # grpcio>=1.48.1 has hanging/crashing issues: https://github.com/grpc/grpc/issues/30843 and https://github.com/grpc/grpc/issues/31885
-        # ensure version we require is >= that with which we generated the grpc code (set in dev-requirements)
-        "grpcio>=1.32.0,<1.48.1",
-        "grpcio-health-checking>=1.32.0,<1.44.0",
+        # grpcio 1.44.0 is the min version compatible with both protobuf 3 and 4
+        "grpcio>=1.44.0,<1.48.1",
+        "grpcio-health-checking>=1.44.0",
```

Comment:
re - the pins here, one thing i'm not sure about: this new min version is what's needed assuming you want things to keep working if you upgrade to protobuf4, but if you leave the installed protobuf version at 3, you can keep it back at grpcio 1.32.0

i.e. these combos are valid after this PR:
grpcio 1.32.0 / protobuf 3.20.0
grpcio 1.48.0 / protobuf 3.20.0
grpcio 1.48.0 / protobuf 4.22.0

So we could technically leave the >=1.32.0 but that could still result in a combo of libraries that satisfies our constraints but doesn't load correctly